lms/envs/production.py

```diff
@@ -1042,3 +1042,11 @@ def get_env_setting(setting):
 
 ################# show account activate cta after register ########################
 SHOW_ACCOUNT_ACTIVATION_CTA = ENV_TOKENS.get('SHOW_ACCOUNT_ACTIVATION_CTA', SHOW_ACCOUNT_ACTIVATION_CTA)
+
+################################# CELERY ######################################
+
+CELERY_IMPORTS = (
+    # Since xblock-poll is not a Django app, and XBlocks don't get auto-imported
+    # by celery workers, its tasks will not get auto-discovered:
+    'poll.tasks',
+) + ENV_TOKENS.get('CELERY_IMPORTS', ())
```

Comment:
Okay, I understand what you are trying to do. However, I think this approach is not the best cause:
1. If someone wanted to add something else to CELERY_IMPORTS in common.py, they would have to modify both common.py and production.py
2. What if someone wanted to completely overwrite CELERY_IMPORTS?

I don't have a suggestion that takes take of both of those issues, but these are the possiblities I can think of:

1. Completely overwrite CELERY_IMPORTS:
`CELERY_IMPORTS = ENV_TOKENS.get('CELERY_IMPORTS', CELERY_IMPORTS)`

This would allow a deployment to completely control this setting. However, this would mean, a deployment would have to keep track of what is added to CELERY_IMPORTS in common.py and always add it to EVN_TOKENS.

2. Another alternative is:
`CELERY_IMPORTS = tuple({*CELERY_IMPORTS, *ENV_TOKENS.get('CELERY_IMPORTS', ())`

This would not overwrite additions in common.py. But this doesn't give a deployment full control either.

I favor the 2nd approach.

Reply:
Eitherway, a comment should be written pointing to setting in common.py